packages/react/src/FileList/FileListItem.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { DocumentIcon } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import { formatFileSize } from '../common/formatFileSize'
+import { formatFileType } from '../common/formatFileType'
+
+export type FileListItemProps = {
+  file: File
+  onRemove?: () => void
+} & HTMLAttributes<HTMLLIElement>
+
+export const FileListItem = forwardRef(
+  ({ file, onRemove, className, ...restProps }: FileListItemProps, ref: ForwardedRef<HTMLLIElement>) => (
+    <li {...restProps} ref={ref} className={clsx('ams-file-list__file', className)}>
+      <div className="ams-file-list__file-preview">
+        {file.type.includes('image') ? (
```

Comment:
Is this a better expression of what we mean here? An `includes` can result in false positives for edge cases.

```suggestion
        {file.type.startsWith('image/') ? (
```